### ISSUE TITLE
evaluation metric - 6.566 on ISTD in Table 8

### ISSUE THREAD

**Issue by nachifur** [jiupinjia/Deep-adversarial-decomposition#6]
![image](https://user-images.githubusercontent.com/23144348/97562173-2d848c80-1a1c-11eb-9134-a28da4d726cc.png)
This work is very meaningful.

I tested the code provided by your github on the ISTD dataset. The indicators are as follows. 
Dataset: istd, average rmse-shadow: 8.4986, rmse-non-shadow: 6.3858, rmse-all: 6.7246. 
This is different from the results provided in Table 8. Can you explain what the problem is? 
The way I run the code from [https://github.com/jiupinjia/Deep-adversarial-decomposition](url).
```
python train.py --dataset istd --checkpoint_dir checkpoints --vis_dir val_out --max_num_epochs 200 --batch_size 2 --enable_d1d2 --enable_d3 --enable_synfake --net_G unet_256 --pixel_loss pixel_loss --metric labrmse_gt1
python eval_deshadow.py --dataset istd --ckptdir checkpoints --net_G unet_256 --in_size 256 --save_output
```
The dataset download from https://github.com/DeepInsight-PCALab/ST-CGAN.
The txt file is log file (training+evl).
[nohup.txt](https://github.com/jiupinjia/Deep-adversarial-decomposition/files/5457960/nohup.txt)
I will appreciate any reply from you.

**Comment by jiupinjia** [comment]
Hi, thanks for your feedback. Please note that when we reported the final accuracy in this table, we were using the Matlab evaluation code provided by the ARGAN and ST-CGAN authors. We found there may be a slight difference in their scores between the python and Matlab scripts. We therefore finally followed the results of the Matlab script in our paper for a fair comparison. Sorry for the inconvenience caused. You may ask the authors for their evaluation source code.

**Comment by nachifur** [comment]
Thank you for your prompt reply. If it is convenient, can I send the evaluation code to my mailbox? 
liujiawei@sia.cn
I will sincerely thank you for your help.

**Comment by jiupinjia** [comment]
I am not sure whether it is good to distribute the code since I am not the owner of it.

**Comment by nachifur** [comment]
I already get code from https://github.com/DeepInsight-PCALab/ST-CGAN/issues/6. jfwang.cs@gmail.com can provided the code. Sincerely thank you for your reply,

**Comment by jiupinjia** [comment]
Great, good luck!

**Comment by nachifur** [comment]
You patiently answered many of my questions, I am very grateful. So I recommend this project - [Mulimg_viewer]( https://github.com/nachifur/Mulimg_viewer/wiki) to you. Mulimg viewer can easily enlarge the picture in parallel mode. I hope this project of mine can help you in the comparison picture of your paper in the future.

**Comment by jiupinjia** [comment]
Such a great project! Thank you so much for your kind recommendation. It always makes me a headache when I was making a zoom-in comparison. With this tool, I believe it would be a great help. I have starred it and you can keep it updating.

**Comment by nachifur** [comment]
I am very happy that Mulimg viewer can help you, I will use my free time to update it in the future.
